main/ZsensorHCSR501.ino

```diff
@@ -65,7 +65,8 @@ void MeasureHCSR501() {
     digitalWrite(HCSR501_LED_NOTIFY_GPIO, pirState == HCSR501_LED_ON);
 #  endif
     HCSR501data["origin"] = subjectHCSR501toMQTT;
-    handleJsonEnqueue(HCSR501data);
+    if (HCSR501data.size() > 1)
+      handleJsonEnqueue(HCSR501data);
```

Comment:
HCSR501data["origin"] = subjectHCSR501toMQTT; could be moved inside the condition as it does not need to be done if the size = 0

Reply:
Agree, thanks, done :-)